pom.xml

```diff
@@ -629,17 +592,16 @@
     </reporting>
     <profiles>
         <profile>
-            <id>pmd</id>
+            <id>quickbuild</id>
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
-            <build>
-                <plugins>
-                    <plugin>
-                        <artifactId>maven-pmd-plugin</artifactId>
-                    </plugin>
-                </plugins>
-            </build>
+            <properties>
```

Comment:
This might be some weirdness with Maven, so I'm actively trying to find some documentation on this, but I can't seem to put a `property` element inside of the `properties`. Inside the activation block, it's different because if I wanted to require multiple properties, the items inside of the activation block are `AND`ed together so I would just define each one as a `property` element. 
As far as meaning goes, there is just a slight difference here. 
In the properties for the profile, I am *setting* these as part of the profile.
In the activation block, I am defining the condition that must be true for activation to happen. (e.g. *skipDocs must not be true*) Hope that makes sense!